java/src/jmri/jmrit/throttle/ThrottlesPreferences.java

```diff
@@ -51,11 +52,11 @@ public ThrottlesPreferences() {
             root = null;
         }
         if (root != null) {
-            load(root.getChild("throttlesPreferences"));
+            setXml(root.getChild("throttlesPreferences"));
         }
     }
 
-    public void load(org.jdom2.Element e) {
+    public void setXml(org.jdom2.Element e) {
```

Comment:
We use load() and store() thru all of JMRI so I think it would be good if these methods keep the names load() and store().
I agree that setXml() and getXml() might be better names, but load() and store() are the names commonly used in JMRI for this.

Reply:
done.